_drafts/rails-migrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Rails Migrations
+---
+
+We had a lively conversation today at [Big Nerd Ranch][bnr] about Rails migrations.
+What are they really for?
+How long should you keep them around?
+What about this crazy schema.rb file?!
+TIL WE HAZ OPINIONS!
+
+⚠️ The views of this post are my own.
+This is in no away a representation of our team's view of the subject.
+Quite the contrary, actually.
+
+## The Migration Files
+
+Migrations are an extremely powerful tool used to apply small transformations to your app's database structure.
+This helps you keep all the databases involved in the development of your app in sync:
+* your development environment
+* your test environment
+* your coworker's ⤴️
+* staging env(s)
+* production
+
+With that, you are able to avoid a lot of the pain of managing database structure across your team and deployment.
+
+### Can I delete them?
+
+It depends (doesn't it always).
+Test Double says, ["Keep them working"][td].
```

Comment:
It's a bit confusing that you say "it depends" here and then only give one view at first. The other view isn't given until line 57.

I would lean more towards ordering the post like this:

1. Introduce the question (can I delete migration files?)
2. Define concepts (migration files, schema file)
3. Give different views (Rails and Test Double)
4. Work out the implications of the different views, and how they lead you to a different answer on the question

Whereas you introduce one view, dive into defining the concepts, then go to the other view. If your order is intentional, would love to hear your thoughts on it!